cmake/modules/Finduuid.cmake

```diff
@@ -34,16 +34,18 @@ if(NOT UUID_INCLUDE_DIR)
   find_path(UUID_INCLUDE_DIR uuid/uuid.h)
 endif()
 
-if(EXISTS UUID_INCLUDE_DIR)
+if(EXISTS "${UUID_INCLUDE_DIR}")
   set(UUID_INCLUDE_DIRS ${UUID_INCLUDE_DIR})
   set(CMAKE_REQUIRED_INCLUDES ${UUID_INCLUDE_DIRS})
   check_cxx_symbol_exists("uuid_generate_random" "uuid/uuid.h" _uuid_header_only)
 endif()
 
-if(NOT _uuid_header_only AND NOT UUID_LIBRARY)
+if(NOT _uuid_header_only)
   check_library_exists("uuid" "uuid_generate_random" "" _have_libuuid)
```

Comment:
This should probably be inside the `if(NOT UUID_LIBRARY)`, so it's not checked if `UUID_LIBRARY` is already set.